server/routes/message/view-group-message.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const { HttpError } = require('../../utils/errors');
+const { Message } = require('../../models');
+const { authenticateToken } = require('../../utils/token-helpers');
+
+const router = express.Router();
+router.get('/:id/messages', authenticateToken, async (req, res) => {
+    const { id } = req.user;
+    const groupId  = Number(req.params.id);
+
+    try {
+        const message = new Message(id,groupId);
+        await message.view();
+        return res.status(200).json({ message: 'Viewing Messages' });
```

Comment:
This should return the actual messages for the group
```suggestion
        return res.status(200).json({ message:  messages });
```

Reply:
Thanks